systers_portal/templates/meetup/snippets/meetup_sidebar.html

```diff
@@ -0,0 +1,6 @@
+<div class="sidebar-module mb40 btn-group-vertical" role="group">
+  <p><a href="{% url 'about_meetup_location' object.slug%}">About</a></p><br>
+  <p><a class="btn btn-success btn-block mb15" href="#" role="button">New Meetup</a></p><br>
+  <p><a class="btn btn-success btn-block mb15" href="#" role="button">Add Memners</a></p><br>
```

Comment:
There is a typo in "Memners". Also should be singular.
This shouldn't be a button. Neither "New Meetup". Aggregate them into a div "Meetup Location Actions". We have discussed that.

Also think for future. If you have a possibility to add a member, what is your strategy to remove them from the members?